app/models/user.rb

```diff
@@ -60,7 +60,7 @@ def enqueue_inactive_email
   end
 
   def able_to_edit_repo?(repo)
-    repo.user_name == github
+    repo.user_name == github.downcase
```

Comment:
When I checked the code and what's being stored in the database, I observed that the repo.user_name is extracted from github and I think it's already converted to downcase. You might want to double check it to confirm.